letsencrypt/config.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-version: 5.0.26
+version: 5.0.27
```

Comment:
Since there has been a older PR merged, this needs rebasing.

Also this seems a bit bigger more generic feature addition, so I'd suggest we go to 5.1.0 for this.